src/components/Recipes/AddToMealplanContainer.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { AddRecipeTitleToMealplanToggle } from '../Button'
+import imagePathPlus from '../../images/inverted-plus.svg'
+import { v4 as uuidv4 } from 'uuid'
+
+import styled from 'styled-components'
+import { saveToLocal, loadFromLocal } from '../../utils/localStorage'
+
+export default function AddToMealplanContainer({ title }) {
```

Comment:
Ich glaube AddToMealplan reicht :)

Reply:
Wurde auch in der Umbenennungsaktion berücksichtigt. :)